### DISCUSSION TITLE
Using Webdriver V7 after chrome update to version 115

### DISCUSSION THREAD

**Discussion by haik-ham** [webdriverio/webdriverio#10854] [Q&A] [answered]
Hi Team,

 I am facing same issues described in [#10806](https://github.com/webdriverio/webdriverio/discussions/10806) but I'm using webdriver V7 due to some node related restrictions in my project. I also saw that the issue has being fixed in [#10767](https://github.com/webdriverio/webdriverio/pull/10767) PR. I want to ask if there is some workaround for V7 or the only option is to update to v8?
 
Errors:
ERROR webdriver: Request failed with status 500 due to unknown error: unknown error: cannot find Chrome binary
ERROR webdriver: unknown error: unknown error: cannot find Chrome binary

**Comment by christian-bromann** [comment, marked as answer]
You can set the Chrome binary via:

```js
{
    browserName: 'chrome',
    'goog:chromeOptions': {
        binary: '...'
    }
}
```

Alternatively you can raise a PR to help port this feature into the `v7` branch or update to v8.
